package/network/utils/comgt/files/ncm.sh

```diff
@@ -164,9 +184,10 @@ proto_ncm_teardown() {
 	local device
 	json_get_vars device
 
-	echo "Stopping network"
+	echo "Stopping network device:$device"
 
 	manufacturer=`gcom -d "$device" -s /etc/gcom/getcardinfo.gcom | awk '/Manufacturer/ { print tolower($2) }'`
+	manufacturer=`gcom -d "$device" -s /etc/gcom/getcardinfo.gcom | sed '2q;d' | tr -d "\n\r" | tr [A-Z] [a-z]`
```

Comment:
same as before, done duplicate and leave the old line in place. please explain the change